client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/style.scss

```diff
@@ -1,4 +1,6 @@
 @import '@wordpress/base-styles/variables';
+@import '@wordpress/base-styles/breakpoints.scss';
```

Comment:
No need for the `.scss` part, just `@wordpress/base-styles/breakpoints` will do 🙂